scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPipelineModel.scala

```diff
@@ -33,10 +33,7 @@ import scala.util.Random
 
 class H2OMOJOPipelineModel(override val uid: String) extends H2OMOJOModelBase[H2OMOJOPipelineModel] {
 
-  @transient private lazy val mojoPipeline: MojoPipeline = {
-    val reader = MojoPipelineReaderBackendFactory.createReaderBackend(new ByteArrayInputStream(getMojoData()))
-    MojoPipeline.loadFrom(reader)
-  }
+  H2OMOJOPipelineCache.startCleanupThread()
```

Comment:
I expect this will launch 4 threads if `--executor-cores=4`.

Reply:
Can you think of a solution that will utilize an existing thread from a shared thread pool? I'm afraid that java `Timer` or `ScheduledThreadPoolExecutor` always create a new thread...